src/controllers/debts.ts

```diff
@@ -80,6 +80,70 @@ export const deleteOne = async (req: Request, res: Response) => {
   }
 };
 
+const calculateSumByTpe = async (userId: string, isIncome: boolean) => {
+  try {
+    const debtsAgg = await debtModel.Debt.aggregate([
+      {
+        $match: {
+          userId: new ObjectId(userId),
+        },
+      },
+      {
+        $group: {
+          _id: null,
+          sum: {
+            $sum: {
+              $cond: [
+                isIncome,
+                {
+                  $cond: [{ $eq: ["$type", "loan"] }, "$amount", 0],
```

Comment:
I think you could change the $type value instead of having the whole statement inside a ternary operation
```suggestion
                  $cond: [{ $eq: ["$type", isIncome ? "loan": "debt"] }, "$amount", 0],
```